src/Util/MIMETypeUtil.m

```diff
@@ -13,6 +13,7 @@
 NSString *const OWSMimeTypeApplicationOctetStream = @"application/octet-stream";
 NSString *const OWSMimeTypeImagePng = @"image/png";
 NSString *const OWSMimeTypeOversizeTextMessage = @"text/x-signal-plain";
+NSString *const OWSMimeTypeUnknownForTests = @"unknown/mimetype";
```

Comment:
In the Debug UI, there will soon be an action to send an attachment with this MIME type, which should always be "unknown" to the recipient.